server/routes/userRouter.js

```diff
@@ -178,4 +178,20 @@ router.get('/', auth, async (req, res) => {
         });
 });
 
+router.post('/getone', auth, async (req, res) => {
```

Comment:
```suggestion
router.post('/getUserProfile', auth, async (req, res) => {
```

may be a better name for this endpoint :)